### PR TITLE
Adding Primitive POS with minimal features

### PR DESCRIPTION
**Pull request by nurdtechie98**
Added a primitive POS system with item selector and invoice calculation feature and integrated it in the accounting package.

### CODE REVIEW

**Comment by rmehta** [comment]
Share an animate GIF on how this works!

File: src/components/POS.vue [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import frappe from "frappejs";
+
+export default {
+  components: {
+    Transaction,
+    ItemList
+  },
+  data() {
+    return {
+      items: [
```
Comment (rmehta) [inline src/components/POS.vue]: query this with `frappe.db.getAll`

File: src/components/Transaction.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    <td>
+                        <span v-if="!item.editing" @dblclick="toggleEdit(item)">{{ item.numberOfItems }}</span>
+                        <input v-if="item.editing" @blur="toggleEdit(item)" type="number" v-model="item.numberOfItems">
+                    </td>
+                    <td>{{ item.numberOfItems * item.item.price }}</td>
+                    <td><a class="fa fa-times" @click="removeItem(item)">X</a></td>
+                </tr>
+            </tbody>
+        </table>
+        <p v-if="!items.length">No items have been added.</p>
```
Comment (rmehta) [inline src/components/Transaction.vue]: Add a "Make Invoice" button that will insert the items in the `Invoice` model, lets just make an invoice in the first cut, worry about payments later

**Review by rmehta — CHANGES_REQUESTED** [review]

**Comment by karan28598** [comment]
Integrated DB with list items and added invoice generation.
@rmehta, the features have been added.

**Comment by karan28598** [comment]
Integrated DB with list items and added invoice generation.
@rmehta, the features have been added. 
pos

**Comment by nurdtechie98** [comment]
@rmehta all changes asked have been done, let us know for more changes and additions.

**Comment by nurdtechie98** [comment]
error alert messages added temp invoice object created

File: src/App.vue [hunk 3]
```diff
@@ -4,21 +4,47 @@
       <router-view />
     </frappe-desk>
     <router-view v-else name="setup" />
+    <frappe-modal ref="modal" :show="showModal" v-bind="modalOptions" @close-modal="showModal = false"/>
```
Comment (netchampfaris) [inline src/App.vue]: rebase with upstream/move-to-vue branch, this file shouldn't be in changed files

File: src/components/Numpad.vue [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    addDecimalPoint: function(value) {
+      if (!this.amount.includes(".")) this.amount += ".";
+    }
+  }
+};
+</script>
+<style scoped>
+.btn {
+    font-size: 20px;
```
Comment (netchampfaris) [inline src/components/Numpad.vue]: use `rem` instead of `px`

File: src/components/pos/POS.vue [hunk 5]
```diff
@@ -0,0 +1,209 @@
+<template>
```
Comment (netchampfaris) [inline src/components/pos/POS.vue]: let's put all POS related components in `src/pages/PointOfSale/`

**Review by netchampfaris — CHANGES_REQUESTED** [review]

**Comment by nurdtechie98** [comment]
@netchampfaris changes asked have been made pls review

**Pull request closed without merging**